src/index.html

```diff
@@ -7,6 +7,316 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav>
```

Comment:
The 'nav' tag should have a class 'header__nav' for semantic and styling purposes.